python/py-pygraphviz/Portfile

```diff
@@ -41,28 +41,12 @@ if {${name} ne ${subport}} {
                     port:py${python.version}-mock \
                     port:py${python.version}-nose
 
-    if {${python.version} == 36} {
-        github.setup    pygraphviz pygraphviz 1.6 pygraphviz-
-        github.tarball_from archive
-        revision        0
-        checksums       rmd160  22cf7a2a8aeef0bdb34a56a0570b38ae4b7f561b \
-                        sha256  9b0a7d9df881c9bb1d597ac72e96227d38b432e948d753dd9df021883668fb0e \
-                        size    101131
-    } elseif {${python.version} in "27 35"} {
-        github.setup    pygraphviz pygraphviz 1.5 pygraphviz-
-        github.tarball_from archive
-        revision        0
-        checksums       rmd160  865d61939646f1a813fbda71e4bcf812af515aee \
-                        sha256  58c3122c7efb9a62275a915953818acd5b3e113b943075501d316ab2291dc21d \
-                        size    102796
-    } else {
-        depends_test    port:py${python.version}-pytest
-
-        test.cmd        py.test-${python.branch}
-        test.target
-        test.args       --pyargs pygraphviz
-    }
-
+    depends_test    port:py${python.version}-pytest
```

Comment:
this will overwrite the test-dependencies that are added above, that's probably not the intention... or?

Reply:
My intention is to strip out the blocks specifically for py27, py35, and py36. From what I can tell, those lines for test capabilities were contingent on later python versions (37+). Now that that set of conditionals are removed, I moved the test block back one nest level, so they should still work. Is that not right?

Reply:
that depends, if the test indeed only need `pytest` then this will technically work. But it would be cleaner then to just remove the block
```
    depends_test-append \
                    port:py${python.version}-doctest-ignore-unicode \
                    port:py${python.version}-mock \
                    port:py${python.version}-nose
```
completely instead of overwriting it

Reply:
Oh, I see what you are pointing out. I was quickly editing old code that seems to have been a bit inconsistent. Any objection to deleting all the lines related to testing? I don't have the knowledge or interest in maintaining it.

Reply:
I'd prefer to keep the testing part in - if you don't mind I'll take a look at it shortly and can push to your branch to update the PR

Reply:
Don't mind at all. Thanks for reviewing and glad to have your edits.